packages/components/src/list/list-item-collapse/style.scss

```diff
@@ -0,0 +1,29 @@
+.woocommerce-list__item.list-item-collapse {
+	justify-content: space-between;
+
+	p {
+		margin: 0;
+	}
+
+	.list-item-collapse__icon-container {
+		justify-content: flex-end;
+	}
+}
+
+.list-item-collapse-enter {
+	max-height: 0;
+}
+.list-item-collapse-enter-active {
+	max-height: 100vh;
+	transition: max-height 500ms ease-out;
+	overflow: hidden;
+}
+
+.list-item-collapse-exit {
+	max-height: 100vh;
+}
+.list-item-collapse-exit-active {
+	max-height: 0;
+	transition: max-height 500ms;
```

Comment:
I'm not sure if this is what's causing it.. but the collapsing transition seems to pause before animating, where the expansion transition does not.

Reply:
Yeah, just playing around with that now, I guess it is because of the 100vh, so it takes a little while before getting to the last 200px. I need to find a better way for this.